src/main/java/com/kuflow/rest/client/resource/ProcessElementValueItemResource.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2021-present KuFlow S.L.
+ *
+ * All rights reserved.
+ */
+
+package com.kuflow.rest.client.resource;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@JsonSerialize(using = ProcessElementValueItemResourceSerializer.class)
+@JsonDeserialize(using = ProcessElementValueItemResourceDeserializer.class)
+public class ProcessElementValueItemResource {
+
+    private Object value;
+
+    private ProcessElementValueItemResource(Object value) {
+        this.value = value;
+    }
+
+    public static ProcessElementValueItemResource of(String value) {
+        return new ProcessElementValueItemResource(value);
+    }
+
+    public static ProcessElementValueItemResource of(Double value) {
+        return new ProcessElementValueItemResource(value);
+    }
+
+    Object getRawValue() {
+        return this.value;
+    }
+
+    public String getValueAsString() {
+        if (this.value == null) {
+            return null;
+        }
+
+        if (this.value instanceof String) {
+            return (String) this.value;
+        }
+
+        return String.valueOf(this.value);
+    }
+
+    public Double getValueAsDouble() {
+        if (this.value == null) {
+            return null;
+        }
+
+        if (this.value instanceof Number) {
+            return ((Number) this.value).doubleValue();
+        }
+
+        return Double.valueOf(this.value.toString());
+    }
```

Comment:
I think that we need to add another 'getValueAsXXX' method for dates. Now we can have the following field types: Text, Number or Date, ie:

getValueAsInstant ¿?

Reply:
Is only getValueAsDate() -> LocalDate
and getValuesAsDate()-> List<LocalDate>
?

Reply:
Are we storing the TimeZone?

If not, you are right that we must use LocalDate.

Maybe I think that is better to use `getValueAsLocalDate`.

Also, we need to add all the `X.of(LocalDate value)`, we need to add the methods for both sides, to read and create.

Reply:
An relataded methos and test

Reply:
yes, all that stuffs